e2e-tests/playwright/tests/functional/system_console/desktop_landing.spec.ts

```diff
@@ -0,0 +1,75 @@
+import {test, expect, chromium, Page} from '@playwright/test';
+
+// Helper function to intercept API request and modify the response
+async function interceptConfigWithLandingPage(page: Page, enabled: boolean) {
+    const apiUrl = '**/api/v4/config/client?format=old**';
+    await page.route(apiUrl, (route) => {
+        route.fulfill({
+            status: 200,
+            body: JSON.stringify({
+                EmailLoginButtonBorderColor: '#2389D7',
+                EmailLoginButtonColor: '#0000',
+                EmailLoginButtonTextColor: '#2389D7',
+                EnableDesktopLandingPage: enabled,
+                EnableSignInWithEmail: 'true',
+                EnableSignInWithUsername: 'true',
+                EnableSignUpWithEmail: 'true',
+                SiteName: 'Mattermost',
+                SiteURL: 'http://localhost:8065',
+            }),
+            headers: {'Content-Type': 'application/json'},
+        });
+    });
+}
+
+test('MM-T5640_1 should not see landing page ', async () => {
+    const browser = await chromium.launch();
+    const context = await browser.newContext();
+    const page = await context.newPage();
+    await interceptConfigWithLandingPage(page, false);
+
+    // Navigate to your starting URL
+    await page.goto('http://localhost:8065');
+
+    // Wait until the URL contains '/login'
+    await page.waitForURL(/.*\/login.*/);
+
+    // At this point, the URL should contain '/login'
+    expect(page.url()).toContain('/login');
+
+    await page.waitForLoadState('networkidle');
+    await page.waitForLoadState('domcontentloaded');
```

Comment:
Do we need these when we're waiting for the `#saveSetting` button below?

Reply:
Yes @hmhealey. This is a better way to wait for intercepted API to return respose and then see the affects of it on UI.
In a non-intercepted use case, just waiting for Savebutton would be enough.